addons/mail/static/src/components/message/message.js

```diff
@@ -414,22 +269,26 @@ export class Message extends Component {
         if (!this.message) {
             return;
         }
-        if (this._prettyBodyRef.el && this.message.prettyBody !== this._lastPrettyBody) {
+        
+        if (this._authorRef.el) {
+            this._authorRef.el.innerHTML = this.message.processDomTransformation(this.message.author.nameOrDisplayName);
```

Comment:
xss?

Reply:
I'll double check, but all the data should originate from the server.

Reply:
So, you still can't trust it. Rename your user `<img src=1 onerror="alert('oups')"/>` and send a message to the admin user. Same for the subject. See the warning at https://www.odoo.com/documentation/14.0/developer/reference/addons/security.html#escaping-vs-sanitizing